File: src/medrelay/enums/user_role.py
from enum import StrEnum


class UserRole(StrEnum):
    ADMIN = "ADMIN"
    CONTROLLER = "CONTROLLER"
    READ_ONLY = "READ_ONLY"

    def label(self) -> str:
        return {
            UserRole.ADMIN: "Admin",
            UserRole.CONTROLLER: "Controller",
            UserRole.READ_ONLY: "Read Only",
        }[self]

File: app/Enums/UserRole.php
<?php

declare(strict_types=1);

namespace App\Enums;

enum UserRole: string
{
    case Admin = 'admin';
    case Controller = 'controller';
    case ReadOnly = 'read_only';

    public function label(): string
    {
        return match ($this) {
            self::Admin => 'Admin',
            self::Controller => 'Controller',
            self::ReadOnly => 'Read Only',
        };
    }
}
